### ISSUE TITLE
[Accessibility]: add command to focus notifications through the command palette

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36753]
### Audit type

Keyboard navigation

### User journey audit issue

#34054

### Problem description

On the search page with the `search-export` extension enabled, click on the `Sourcegraph: Export search results` action. Observe that a notification pops us with a CSV download link. This link is near the end of the tab order, and due to our infinite scrolling search results list it takes *really* long to focus the link.

### Expected behavior

An idiom in editors (like VS Code) is exposing a command (in the keyboard navigable command palette) to focus elements that are out-of-the-way. We can add a command to our command palette to focus the first notification, from which users can tab to subsequent notifications.

### Additional details

![Screen Shot 2022-06-07 at 5 29 06 PM](https://user-images.githubusercontent.com/37420160/172486013-a5508e3e-fb1a-4f3a-bed1-362ae06e15c1.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
